### PR TITLE
When borrow, take into account the origination fee when calculating LUP

### PR DESCRIPTION
**Pull request by grandizzy**
TODO: rewrite `testMergeOrRemoveCollateral` scenario

### CODE REVIEW

File: src/base/Pool.sol [hunk 1]
```diff
@@ -418,13 +418,14 @@ abstract contract Pool is Clone, ReentrancyGuard, Multicall, IPool {
 
             // add origination fee to the amount to borrow and add to borrower's debt
             uint256 debtChange = Maths.wmul(amountToBorrow_, _feeRate(poolState.rate) + Maths.WAD);
-            borrowerDebt += debtChange;
+            borrowerDebt   += debtChange;
+            poolState.debt += debtChange;
 
             // check that drawing debt doesn't leave borrower debt under min debt amount
             _revertOnMinDebt(poolState.debt, borrowerDebt);
```
Comment (mattcushman) [inline src/base/Pool.sol:424]: Minor point but... first time the borrower borrows, doesn't the loan not yet exist, so we include the new debt in the numerator but not the denominator of the min debt size calculation?  I don't think it matters (would be strictly more conservative change if so) and shouldn';t matter much in the case of small debt sizes, which is the case we actually care about

File: tests/forge/ERC20Pool/ERC20PoolQuoteToken.t.sol [hunk 2]
```diff
@@ -653,7 +653,7 @@ contract ERC20PoolQuoteTokenTest is ERC20HelperContract {
                 from:     _lender,
                 amount:   10_000 * 1e18,
                 index:    4990,
-                newLup:   _priceAt(4551),
+                newLup:   0.140143083210662942 * 1e18,
```
Comment (mattcushman) [inline tests/forge/ERC20Pool/ERC20PoolQuoteToken.t.sol]: It's nice when these are of the form _priceAt(index) imho

**Review by mattcushman — COMMENTED** [review]
only minor comment was on the

File: src/base/Pool.sol (hunk 1 above)
Comment (grandizzy) [reply src/base/Pool.sol:424]: changed with https://github.com/ajna-finance/contracts/pull/457/commits/d13d9ac39821ac65c349058296723efa7e481334

File: tests/forge/ERC20Pool/ERC20PoolQuoteToken.t.sol (hunk 2 above)
Comment (grandizzy) [reply tests/forge/ERC20Pool/ERC20PoolQuoteToken.t.sol]: change with https://github.com/ajna-finance/contracts/pull/457/commits/d13d9ac39821ac65c349058296723efa7e481334

**Review by mattcushman — APPROVED** [review]
lgtm

**Pull request merged**